client/lib/api/content/content_store.dart

```diff
@@ -134,5 +134,10 @@ abstract class _ContentStore with Store implements Updateable {
         symptomPoster = v;
       })
     ]);
+
+    unawaited(
```

Comment:
I had to read up on unawaited but this looks right. I think it's worth adding a log here.